.prow.yaml

```
@@ -246,6 +246,49 @@ presubmits:
                 path: id_rsa
               - key: id_rsa.pub
                 path: id_rsa.pub
+  - name: e2e-oai-fedora-34
+    annotations:
+    labels:
+    run_if_changed: '^e2e/|.prow.yaml'
```

This changes won't be necessary in the final version
```suggestion
    run_if_changed: '^e2e/'
```